components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -1915,12 +1918,54 @@ public static String addUsernameToToken(AuthenticatedUser authenticatedUser, Str
     }
 
     /**
-     * Get authorized user from the {@link AccessTokenDO}. When getting authorized user we also make sure flag to
-     * determine whether the user is federated or not is set.
+     * Validates the json provided.
      *
-     * @param accessTokenDO
-     * @return
+     * @param redirectURL
```

Comment:
If you are adding doc comments, you need to fill the details as well :). Otherwise release build will fail when we enable java doc generation in Java 8.